WcaOnRails/app/controllers/registrations_controller.rb

```diff
@@ -66,7 +66,7 @@ def psych_sheet_event
 
   def index
     @competition = competition_from_params
-    @registrations = @competition.registrations.accepted
+    @registrations = @competition.registrations.accepted.includes(:user, :registration_events)
```

Comment:
Because of calls to `Registration#wca_id` which then call `user.wca_id`: https://github.com/cubing/worldcubeassociation.org/blob/master/WcaOnRails/app/views/registrations/index.html.erb#L32-L34